pennylane/_grad.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the autograd wrappers :class:`grad` and :func:`jacobian`
+"""
+import numpy as _np
+
+# pylint: disable=unused-import
+from autograd.core import make_vjp as _make_vjp, make_jvp as _make_jvp
```

Comment:
```suggestion
from autograd.core import make_vjp as _make_vjp
```
No need to disable this one, we can simply remove the unused import :)

Reply:
Oh true. I was wondering why this was complaining (since it looked like it was being used), but I apparently just got confused with the names with/without `_` and `vjp` or `jvp`. :laughing: